src/pyflex/config.py

```diff
@@ -271,7 +281,7 @@ def _convert_to_array(self, npts):
         """
         attributes = ("stalta_waterlevel", "tshift_acceptance_level",
                       "dlna_acceptance_level", "cc_acceptance_level",
-                      "s2n_limit")
+                      "s2n_limit", "c_1")
```

Comment:
At that point it might be a good idea to turn all the `c_` parameters into arrays?